### DISCUSSION TITLE
[Suggestions] Hero screen, number of creatures, font size too large

### DISCUSSION THREAD

**Discussion by LeHerosInconnu** [ihhub/fheroes2#4919] [General]
In fheroes2, the font size for the number of creatures is way too large.
It hurts my eyes. :)

I prefer the font size used in the original game, the number of creatures is much more discreet.

In original game:

![Hero screen H2 10](https://user-images.githubusercontent.com/43583994/93109464-60650200-f6b4-11ea-8072-01075ade6bdf.png)

In fheroes2:

![Hero screen fh2 10](https://user-images.githubusercontent.com/43583994/93109527-78d51c80-f6b4-11ea-8ecf-6332a9444669.png)

**Comment by Branikolog** [comment]
In my opinion larger numbers look better.
Considering the fact that currently our game supports higher resolutions,  little numbers would be much less friendly to eyes of players...

**Comment by LeHerosInconnu** [comment]
@Branikolog.

> In my opinion larger numbers look better.
> Considering the fact that currently our game supports higher resolutions, little numbers would be much less friendly to eyes of players...

Personally, even with a resolution of 1920 x 1200 pixels, I do not find this to be unreadable at real size of use (on a 24 inch screen).
Look at the numbers and texts in the areas circled in red (if you have a large enough screen).

The large font size can also be an option (and can even be automatically activated from a certain resolution), notably in the area to the right of the screen for the numbers of resources and creatures (especially for small screens).
But I still prefer the smaller font size for the number of creatures in the hero's army. :)

In fheroes2, resolution of 1920 x 1200 pixels:

![Screen fh2 1920 1200 cercle 010](https://user-images.githubusercontent.com/43583994/93236561-47755300-f77f-11ea-9f23-c6115a7071ff.png)

**Comment by ihhub** [comment]
I'm making this issue as discussion till we find a solution / agreement about this change.
